content/en/admin/optional/sso.md

```diff
@@ -12,3 +12,65 @@ This page is under construction.
 For the moment, please refer to [this pull request](https://github.com/mastodon/mastodon/pull/16221)
 {{< /hint >}}
 
+# OmniAuth
+Mastodon supports the CAS, SAML and OpenID Connect protocols for external authentication.
+These can either be configured in addition to native logins on Mastodon or as the sole
+identity provider using `OMNIAUTH_ONLY=true`.
```

Comment:
Also needs docs for `ONE_CLICK_SSO_LOGIN` currently this is hiding under CAS at the bottom of the page, when these two options are usually used together.

And there's `SSO_ACCOUNT_SETTINGS` and `SSO_ACCOUNT_SIGN_UP`